pkg/utils/cluster.go

```diff
@@ -26,6 +28,14 @@ const (
 	controlPlaneNodeLabelKey = "node-role.kubernetes.io/control-plane"
 )
 
+var (
+	oneNode     = intstr.FromInt32(1)
+	defaultNpcl = &sriovnetworkv1.SriovNetworkPoolConfig{Spec: sriovnetworkv1.SriovNetworkPoolConfigSpec{
```

Comment:
nit: can we use full name here ? e.g defaultPoolConfig ?

also the 'l' at the end is not related